website/src/pages/showcase.tsx

```diff
@@ -18,8 +19,13 @@ const renderApp = (app, i) => <AppBox app={app} key={`app-${app.name}-${i}`} />;
 function Section({
   element = 'section',
   children,
-  className,
   background = 'light',
+  className,
+}: {
+  element?: any;
```

Comment:
Can this prop be done away with? Looks like no usages in the file and we can default to using `<section>`